ppr-ui/src/components/search/SearchBarList.vue

```diff
@@ -130,9 +133,10 @@ export default defineComponent({
         }
         return allSearchTypes
       }),
+      isPPROnly: computed((): boolean => hasPprRole.value && !(hasMhrRole.value && getFeatureFlag('mhr-ui-enabled'))),
       displayItems: [],
       displayGroup: {
-        1: false,
+        1: !(hasMhrRole.value && hasPprRole.value),
```

Comment:
When looking i found neither `hasMhrRole` or `hasPprRole` was set when this component mounted (the timing i was referring to previously). 
Does this not always initialize `displayGroup` as `1: true` due to that?
If it works though, great 👍

Reply:
I found a scenario when it doesn't work, so need to re-think this.

Reply:
As we discussed previously, a watcher could help with the timing.